Documentation/resources.md

```diff
@@ -55,6 +55,43 @@ Expose the user-defined Alertmanager web server within the cluster on the follow
 
 Expose the Alertmanager web server within the cluster on the following ports:
 * Port 9094 provides access to all the Alertmanager endpoints. Granting access requires binding a user to the `monitoring-alertmanager-view` role (for read-only operations) or `monitoring-alertmanager-edit` role in the `openshift-monitoring` project.
+```
+# monitoring-alertmanager-view grants read permissions.
```

Comment:
I assume comments from @eromanova97 apply also here and overall in any comment.

Reply:
Yes, that would be great. I just don't remember what markdown can and cannot do compared to asciidoc, so it might need slight adjustments :+1:

Reply:
Yep, for now the generated examples aren't included in the docs, I'll improve the ascii formatting once we start taking them into account.